### DISCUSSION TITLE
Nerd v2 only hashing 55kh/s

### DISCUSSION THREAD

**Discussion by Russty123** [BitMaker-hub/NerdMiner_v2#404] [Q&A]
I have the 2.8” screen ESP32_2432028… its only hashing the v1 speed. Am i missing something? I have flashed it a few times.

**Comment by IAmSixNine** [comment]
The 4 i have with 2.4 inch screens also mine at 55kh/s. I just figured this was normal. My TTGO t3 does 75kh/s

**Reply by Russty123** [reply]
Doesnt seem right , it might be a hardware issue? 

Or the update software hasnt been updated for version 2 for this device??

**Reply by IAmSixNine** [reply]
Not sure. but it would be cool to see if they can go from 55kh/s to 75Kh/s. 
I also have 3 of the no display esp32-wrom units and they also all do 55kh/s
Currently the only device i have that does 75kh/s is the TTGO ts3. I have tried several different times to flash/reflash and then erasing and flashing but i get the same results.

**Reply by Russty123** [reply]
Be good if someone could reply on why this is so.

**Reply by lupusworax** [reply]
Same issue. I tried to add a heat sink to the ESP32 + aktive cooling  but it is barely warm to the touch, could this be some sort of an exclusive kind of thing for the bought  variant with the fan?

**Reply by IAmSixNine** [reply]
I dont think these get warm enough to warrant a fan. My TS3 does 75Kh/s and is in a plastic case and is barely above room temperature.
